src/module/vue/components/buttons/btn-compendium.vue

```diff
@@ -5,9 +5,8 @@
     @click="openCompendium"
     aria-haspopup="dialog"
     :disabled="disabled"
-  >
-    <slot></slot>
-  </btn-faicon>
+    ><slot name="default"></slot
+  ></btn-faicon>
```

Comment:
ah! forgot about that. yeah, that's a way to avoid whitespace. if the linebreaks only occur within the HTML tags, then they aren't rendered.

context: i was troubleshooting a `<span>` (conditional on slot content) rendering when no slot content was present, and was trying to figure out if something in the component dependencies was causing it.

in the end, the fix to the problem turned out to be something else entirely, i just forgot to change it back; i'll put that on the TODO.

Reply:
should be resolved in all btn-* elements now. i don't think i touched anything else there but i might be mistaken.